server/src/dash-server.ts

```diff
@@ -12,11 +12,19 @@ import {GitHub} from './github';
 import {ViewerLoginQuery, ViewerPullRequestsQuery} from './gql-types';
 
 export class DashServer {
+  private secrets: {
+    GITHUB_CLIENT_ID: string,
+    GITHUB_CLIENT_SECRET: string,
+  };
```

Comment:
Fine for now, but you can add a type here so you don't have to type it out everytime.